scripts/fuzz.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+set -e
+
+function build_fuzz() {
+  PACKAGES=$1
+  
+  for pkg in $PACKAGES; do
+    pushd fuzz/$pkg
+  
+    for file in *.go; do
+      if [[ "$file" == "fuzz_utils.go" ]]; then
+        continue
+      fi
+  
+      NAME=$(echo $file | sed 's/\.go$//1')
+      echo "Building zip file for $pkg/$NAME"
+      go-fuzz-build -func "Fuzz_$NAME" -o "$pkg-$NAME-fuzz.zip" "github.com/lightningnetwork/lnd/fuzz/$pkg"
+    done
+  
+    popd
+  done
+}
+
+# timeout is a cross platform alternative to the GNU timeout command that
+# unfortunately isn't available on macOS by default.
+timeout() {
+    time=$1
+    $2 &
+    pid=$!
+    sleep $time
+    kill -s SIGINT $pid    
+}
+
+function run_fuzz() {
+  PACKAGES=$1
+  RUN_TIME=$2
+  TIMEOUT=$3
+  PROCS=$4
+  
+  for pkg in $PACKAGES; do
+    pushd fuzz/$pkg
+    DIR=$(pwd)
+  
+    for file in *.go; do
+      if [[ "$file" == "fuzz_utils.go" ]]; then
+        continue
+      fi
+  
+      NAME=$(echo $file | sed 's/\.go$//1')
+      WORKDIR=$DIR/$NAME
```

Comment:
My bad, I misspoke. The directory should be a user-specified base directory like `lnd_fuzz_seeds` and then the name after that. So `lnd_fuzz_seeds/accept_channel`:
https://github.com/Crypt-iQ/lnd_fuzz_seeds/
Otherwise we will have to move it to the lnd directory which could be tedious

Reply:
Couldn't you just copy over the completed files after the run?

Reply:
If we want to make sure that the fuzzers don't crash we should throw our data at it so we should use it during the run

Reply:
Ah, okay. Going to add the base directory parameter then.